test/useFormo.test.ts

```diff
@@ -125,6 +125,32 @@ describe("formo", () => {
     ).toBe(10);
   });
 
+  test("handleSubmit does not refer to stale values", async () => {
+    const { result } = renderHook(() =>
+      useFormo(
+        {
+          initialValues: {
+            city: "Milan",
+          },
+          fieldValidators: constant({}),
+        },
+        {
+          onSubmit: (values) => {
+            expect(values.city).toBe("Rome");
+            return taskEither.of(null);
+          },
```

Comment:
instead of putting the expectation here, you could probably declare onSubmit as a mock function and then run assertions on how it was called. Something like:

```ts
const onSubmit = jest.fn(() => taskEither.of(null));

// ...

{
  onSubmit // pass the mock to `useFormo`
}

// ...

await act(async () => ...)

// run assertions on the mock:

expect(onSubmit).toHaveBeenCalled(# of times)
expect(onSubmit.mock.calls[#][0]).toEqual({ city: "Rome" }); // etc.
```